source/StartConditions.cpp

```diff
@@ -91,3 +152,37 @@ const list<Ship> &StartConditions::Ships() const
 {
 	return ships;
 }
+
+
+
+const std::string &StartConditions::GetName() const
+{
+	return name;
+}
+
+
+
+const std::string &StartConditions::GetDescription() const
+{
+	return description;
+}
+
+
+
+const Sprite *StartConditions::GetSprite() const 
+{
+	return sprite;	
+}
+
+
+
+bool StartConditions::operator==(const StartConditions &other) const 
```

Comment:
Why not define starts like we defined missions? That is,
```
start "<name>"
	...
```
That way starts could be identified by a unique name instead of needing to do this. Doing that would also allow plugins to alter existing starts by redefining them, instead of needing to add a new start.